pubtools/_quay/container_image_pusher.py

```diff
@@ -217,7 +217,7 @@ def copy_multiarch_push_item(self, push_item, source_ml):
                         merge_mls_dest_refs.append(dest_ref)
                 except requests.exceptions.HTTPError as e:
                     # Option 3: Destination tag doesn't exist, no ML merging
-                    if e.response.status_code == 404:
+                    if e.response.status_code == 404 or e.response.status_code == 401:
```

Comment:
I think the code relies on 404 in multiple places in the code. Shouldn't we change it everywhere?

Reply:
You are right. I will update other places. Do you think 404 should be kept or removed?

Reply:
I think it wouldn't hurt to keep it. It only gives 401 when a robot account is used, I think

Reply:
Updated.